<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseBtn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择目录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="folderLE">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>改后缀</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Before</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="suffixBeforeLE"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>After</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="suffixAfterLE"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="suffixBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>391</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
